libvast/vast/view.hpp

```diff
@@ -491,4 +491,20 @@ bool type_check(const type& t, const data_view& x);
 bool evaluate_view(const data_view& lhs, relational_operator op,
                    const data_view& rhs);
 
+/// Converts a view according to the accompanying type. I.e. replaces a view
+/// on an internal representation of a field to the representation as defined
+/// in the type.
+/// @param t The type that describes *x*.
+/// @param x The data view on the internal representation of the value.
+/// @return A data view on the external representation of the value.
+data_view to_canonical(const type& t, const data_view& x);
+
+/// Converts a view according to the accompanying type. I.e. replaces a view
+/// on a value as represented in the type to a view on its corresponding
+/// internal representation.
+/// @param t The type that describes *x*.
+/// @param x The data view on the external representation of the value.
+/// @return A data view on the internal representation of the value.
+data_view to_internal(const type& t, const data_view& x);
+
```

Comment:
Could you make the short description of the two functions different? It's hard to understand their differences based on reading the documentation.